tools/wptrunner/wptrunner/environment.py

```diff
@@ -50,6 +51,76 @@ class TestEnvironmentError(Exception):
     pass
 
 
+class _QueueLogger(StructuredLogger):
+    """Structured logger that has a unique name and puts messages on a queue"""
+    def __init__(self, queue):
+        name = str(uuid.uuid4())
+        self.queue = queue
+        super().__init__(name)
+        assert not self.handlers
```

Comment:
What's the significance of this assert? Is it that `super().__init__(name)` shouldn't initialize `self.handlers`? Something does by the time it's used, but does it matter that it wasn't here?